### ISSUE TITLE
Accessibility: Polymer modal dialogs should make the rest of the UI inert and aria-hidden

### ISSUE THREAD

**Issue by minorninth** [Polymer/polymer#5156]
When a modal dialog or other pop-up appears that visually covers the rest of the web page, two things must happen in order for it to be fully accessible:

* The rest of the content should be hidden using aria-hidden=true on each subtree other than the one containing the dialog.
* Any focusable controls should be disabled or otherwise made inert / non-focusable.

I believe all other web frameworks do this already (e.g. Google's Closure/Wiz, AngularUI, React).

Finally, when the modal is closed, ideally focus should be restored to wherever it was last.

To repro, open this demo and click "Modal Dialog":

  https://www.webcomponents.org/element/PolymerElements/paper-dialog/demo/demo/index.html

Then with a screen reader enabled, use the screen reader keys to navigate the rest of the page. It's very easy to explore content outside of the dialog.

For example, with VoiceOver use Ctrl+Option+Left/Right to navigate and Ctrl+Option+Shift+Up to "escape" the dialog. (This shouldn't be possible.)

**Comment by TimvdLippe** [comment]
This issue was moved to PolymerElements/paper-dialog-behavior#125
